operator/pkg/controllers/master/kubeapiserver.go

```diff
@@ -140,6 +140,16 @@ func apiServerPodSpecFor(controlPlane *v1alpha1.ControlPlane) v1.PodSpec {
 					"--tls-private-key-file=/etc/kubernetes/pki/apiserver/apiserver.key",
 					"--authentication-token-webhook-config-file=/var/aws-iam-authenticator/kubeconfig/kubeconfig.yaml",
 					"--encryption-provider-config=/etc/kubernetes/aws-encryption-provider/encryption-configuration.yaml",
+					"--profiling=false",
```

Comment:
As we are going to use this for testing, why turn this off ? Isn't it better for us to have it turned on for better debugging ?